android/src/main/java/com/swmansion/gesturehandler/react/RNGestureHandlerModule.kt

```diff
@@ -1,6 +1,7 @@
 package com.swmansion.gesturehandler.react
 
 import android.content.Context
+import android.graphics.PointF
```

Comment:
revert

Reply:
Done in https://github.com/software-mansion/react-native-gesture-handler/pull/2156/commits/f4056390b05eea7d82f001fb6e8d4a1b16b4f8e9